src/robusta/core/model/jobs.py

```diff
@@ -39,22 +40,38 @@ class JobStatus(BaseModel):
     failed: int = 0
     succeeded: int = 0
     completion_time: Optional[str]
+    failed_time: Optional[str]
     conditions: List[JobCondition]
 
     @staticmethod
     def from_api_server(job: V1Job) -> "JobStatus":
         job_status: V1JobStatus = job.status
         job_conditions: List[JobCondition] = [
-            JobCondition(type=condition.type, message=condition.message) for condition in (job_status.conditions or [])
+            JobCondition(type=condition.type, message=condition.message)
+            for condition in (job_status.conditions or [])
+            if condition.status.lower() == "true"
```

Comment:
we were passing all the conditions even if they wernt active

Reply:
nice